.github/blunderbuss.yml

```diff
@@ -0,0 +1,7 @@
+# Configuration for the Blunderbuss GitHub app. For more info see
+# https://github.com/googleapis/repo-automation-bots/tree/main/packages/blunderbuss
+assign_prs_by:
+- labels:
+  - samples
+  to:
+  - googleapis/java-samples-reviewers
```

Comment:
Are these up to date?

Reply:
If an issue has a label "samples", one member from https://github.com/orgs/googleapis/teams/api-pubsub will be assigned.